README.md

```diff
@@ -23,11 +23,14 @@ At this stage, `Colourblind8` will focus on developing settings for the three mo
 
 ### Functions Included In This Package
 
-- `set_deutera()`:
-  - This function implements a theme that makes plots accesible to people with deuteranopia. It modifies the colour of geometric objects (points, lines, etc) and the layout of the plot.
-- `set_prota()`
+- `plot_lines()`:
+   inputs: x variable,  y variable(s),  labels = None, palette = None, 
+                title = None, x_lab = None, y_lab = None, legend_title = None
+   output: a matplotlib object
+  - This function implements a method that makes line graphs accesible to people with one of these three colourblind variants; deuteranopia, protanopia or tritanopia. It modifies the colour of each line, the title, legend and axes size as well as the face colour.
+- `plot_scatter(()`
   - This function implements a theme that makes plots accesible to people with protanopia. It modifies the colour of geometric objects (points, lines, etc) and the layout of the plot.
-- `set_trita()`
+- `plot_histogram()`
   - This function implements a theme that makes plots accesible to people with tritanopia. It modifies the colour of geometric objects (points, lines, etc) and the layout of the plot.
   
```

Comment:
Hayley, please add the following to Colourblind8 Readme:

`help(Colourblind8)` for class docstring

`help(Colourblind8.modulename)` for module docstring

This is for users to view instructions on how to use our functions.  Thanks!

Reply:
i added them to the readme!